### DISCUSSION TITLE
Looking forward to sionnaV0.15

### DISCUSSION THREAD

**Discussion by jisangXx** [NVlabs/sionna#144] [Q&A] [answered]
I was very pleasantly surprised to find this library, because my research direction is wireless channel research of ray tracing. Can I ask how soon after that sionna will support ray tracing for scattering and diffraction?

**Comment by merlinND** [comment, marked as answer]
Hello @jisangXx,

Sionna 0.15 has now been released, the new features are listed in https://github.com/NVlabs/sionna/discussions/166 :)
